### PR TITLE
Update saber_shtok_buttons.h file

### PR DESCRIPTION
**Pull request by ShtokyD**
Updated with Color Changing support for both button options, added Preset select backwards for 1-button option.

### CODE REVIEW

File: props/saber_shtok_buttons.h [hunk 1]
```diff
@@ -61,25 +96,34 @@ class SaberShtokButtons : public PropBase {
...
+#endif
       Off();
     }
         return true;
 
-// Force
+// Force and Color Change mode
       case EVENTID(BUTTON_NONE, EVENT_TWIST, MODE_ON | BUTTON_POWER):
+#ifndef DISABLE_COLOR_CHANGE
+        if (accel_.x < -0.15) {
```
Comment (profezzorn) [inline props/saber_shtok_buttons.h]: It's better to use fusor.down().x < -0.15
The down() vector handles noise and rotation better.

File: props/saber_shtok_buttons.h [hunk 2]
```diff
@@ -1,3 +1,31 @@
+// 1 Button:
```
Comment (profezzorn) [inline props/saber_shtok_buttons.h]: These comments are a great addition :)

File: props/saber_shtok_buttons.h (hunk 1 above)
Comment (ShtokyD) [reply props/saber_shtok_buttons.h]: I just used these lines from the original code, I don't know coding :)

Comment (profezzorn) [reply props/saber_shtok_buttons.h]: Guess I'll update it later then.

**Pull request merged**